code/modules/admin/callproc/callproc.dm

```diff
@@ -143,7 +143,7 @@
 				if(isnull(current)) return CANCEL
 				current = text2path(current)
 				if(!ispath(current))
-					to_chat("Inputed a bad path: '[current]'")
+					to_chat(usr, "Inputed a bad path: '[current]'")
```

Comment:
`current` will be null post text2path if it didn't produce a path I believe, might want to keep the raw entered text to print here, otherwise printing it is never going to show anything.